scripts/export_producers_platform_data_to_public_database.sh

```diff
@@ -4,30 +4,48 @@
 
 export PERL5LIB="lib:${PERL5LIB}"
 
-PRODUCERS=(
-org-barilla-france-sa
-org-ferrero-france-commerciale
-org-unilever-france-gms
-org-unilever-france-rhd
-org-nestle-france
-org-panzani-sa
-org-cristalco
-org-materne
-org-garofalo-france
-org-brasseries-kronenbourg
-org-carrefour
-org-lustucru-frais
-org-nestle-waters
-org-kambly
-org-kambly-france
-org-saint-hubert
-org-d-aucy
-org-lea-nature
-org-auchan-apaw
-org-les-mousquetaires
-)
-
-for producer in ${PRODUCERS[@]}
+filter_organizations_that_have_automated_export() {
+    perl -e '
+        use strict;
+        use warnings;
+        use ProductOpener::Data qw/get_orgs_collection/;
+
+        my %producers = map { $_,1 } qw(
+            org-barilla-france-sa
```

Comment:
We probably should remove the hardcoded values, and ensure all those orgs have the checkbox checked

Reply:
Done. But I'm not an admin so I can't see the checkbox in the platform.